docs/src/pages/[platform]/getting-started/create-react-app/examples/BasicExampleCode.tsx

```diff
@@ -0,0 +1,31 @@
+import { Button, Flex, Heading, Image, Text } from '@aws-amplify/ui-react';
+import '@aws-amplify/ui-react/styles.css';
```

Comment:
The reason why I created separate files for `BasicExampleCode` and `BasicExampleDemo` is because this CSS import was throwing an error in the docs. The code works if I remove this line, but since we need the customer to be able to copy/paste this code directly in their app, I found a workaround by creating one file for displaying the copy/paste code, and then another file for actually running the example in the docs. Same goes for `CompleteExampleCode` and `CompleteExampleDemo`. Besides naming, the only real difference between the files is whether this CSS import is included.